### PR TITLE
docs: update RELEASE.md

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add git command

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: admin/RELEASE.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ the existing content.
     ```
 * Vet the **admin/** folders for any removed hidden files (Action deploy scripts
   *do not remove these*)
+  * git diff --name-status origin/master admin/
```
Comment (kenjis) [inline admin/RELEASE.md:58]: @MGatner By the way, what if there is a removed hidden file?

Comment (MGatner) [reply admin/RELEASE.md:58]: Like one beginning with a period? I think Git should still report on those.

Comment (MGatner) [reply admin/RELEASE.md:58]: Ohhh I see what you mean: what do we do if a hidden file is actually removed. I don't recall what I did last time but we can probably just go remove it on the target repo before release.

**Review by MGatner — APPROVED** [review]

File: admin/RELEASE.md (hunk 1 above)
Comment (kenjis) [reply admin/RELEASE.md:58]: Yeah, we would have to delete it directly in that repository. But it's a bit cumbersome.
I don't think there is any chance of hidden files being deleted at this point, though.

**Pull request merged**